docs/visualization/plotting-nd-data.ipynb

```diff
@@ -152,6 +154,112 @@
     "from scipp import plot as pl\n",
     "pl.superplot(d)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Tiled Plotting\n",
+    "Tiled plotting uses a similar API to `plot`; the only difference is items to be plotted need to be put into a list as shown."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "one_d_tiled_plots = [d, d]\n",
+    "tiled_plot(one_d_tiled_plots, projection='1d')"
```

Comment:
So when I tried using the slider, it threw
```
TypeError: update_slice() takes 1 positional argument but 2 were given
```
probably because of the missing `change` argument.

Reply:
In addition, I could not test the `Keep` button, because the sliders did not work, but the dropdown menu which is supposed to let you select which entry you want to control, only listed `Some3Ddata_1`, so could not control the first entry.